src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -883,13 +883,26 @@
   <Target Name="CreatePackage"
           Inputs="$(NuSpecPath)"
           Outputs="$(PackageOutputPath)$(Id).$(Version).nupkg">
+
+    <ItemGroup>
+      <_missingFiles Include="@(PackageFile)" Condition="!Exists(%(FullPath))"/>
+    </ItemGroup>
+
+    <PropertyGroup>
+      <_SkipPack Condition="'$(SkipCreatePackageOnMissingFiles)' == 'true' AND '@(_missingFiles)' != ''">true</_SkipPack>
```

Comment:
Maybe a name like "_skipCreatePackage" would be more consistent and obviously associated with the "SkipCreatePackageOnMissingFiles" property.